fs/copy_linux.go

```diff
@@ -64,6 +64,9 @@ func copyFileInfo(fi os.FileInfo, src, name string) error {
 func copyXAttrs(dst, src string, excludes map[string]struct{}, errorHandler XAttrErrorHandler) error {
 	xattrKeys, err := sysx.LListxattr(src)
 	if err != nil {
+		if err == unix.ENOTSUP {
```

Comment:
Oh! Actually; this is handling an error returned by `sysx`,  which is not a raw stdlib / syscall package, perhaps better to use `errors.Is()` here?

Some discussion with @robmry and @djs55 (we happened to be looking into a similar issue), and there's some discussion if `EOPNOTSUPP` should be handled here; this was related to https://elixir.bootlin.com/linux/v6.10/source/fs/fuse/xattr.c#L23

Reply:
> perhaps better to use errors.Is() here

Makes sense, pushed the change

> there's some discussion if `EOPNOTSUPP` should be handled here

It seems pretty reasonable to me, since the `fuse_getxattr` call would also fail in this use case. LMK and I can add add a check for this condition